tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py

```diff
@@ -40,7 +40,9 @@ def parameters(cls):
             ),
             'softmaxed_probabilities': BoolField(
                 optional=True, default=False, description="Indicator that model uses softmax for output layer "
-            )
+            ),
+            'logits_output': StringField(optional=True, description='Logits output layer name'),
+            'custom_label_map': StringField(optional=True, description='Label map')
```

Comment:
how it should be used?  is it path to file with label map or just string or list of tokens?
please note that as the rule, we use dictionary as label map

Reply:
Please also provide new arguments in documentation (adapters/README.md)

Reply:
The work is in progress now, this is a draft PR. It is planned to be a string, but I think, it wouldn't be difficult to pass a dict as label map.

Reply:
Type of label map changed to dict